dashboard/app/models/script.rb

```diff
@@ -449,9 +449,28 @@ def self.add_script(options, data)
       script_level.save! if script_level.changed?
       script_level
     end
-    script_stages.each do |stage|
+    script_stages.each_with_index do |stage|
       stage.script_levels = script_levels_by_stage[stage.id]
+
+      # Go through all the script levels for this stage, except the last one,
+      # and raise an exception if any of them are a multi-page assessment.
+      # (That's when the script level is marked assessment, and the level itself
```

Comment:
this parenthetical statement seems to be asking to be made a helper method on level